src/scss/cdb-components/shapes/add.scss

```diff
@@ -49,6 +49,10 @@ Description
   width: 16px;
   height: 16px;
   text-align: center;
+  &.is-medium {
```

Comment:
This one shouldn't be nested (sorry, I probably missed before).

Also regarding :before & :after , I think they shouldn't be nested /cc @xavijam